test/model/content_test.dart

```diff
@@ -727,59 +727,56 @@ class ContentExample {
       MathBlockNode(
         texSource: "\\Huge 1\n\\huge 2\n\\LARGE 3\n\\Large 4\n\\large 5\n\\normalsize 6\n\\small 7\n\\footnotesize 8\n\\scriptsize 9\n\\tiny 0",
         nodes: [
-          KatexNode(
+          KatexSpanNode(
             styles: KatexSpanStyles(),
             text: null,
             nodes: [
-              KatexNode(
-                styles: KatexSpanStyles(),
-                text: null,
-                nodes: []),
-              KatexNode(
+              KatexStrutNode(heightEm: 1.6034, verticalAlignEm: null),
+              KatexSpanNode(
                 styles: KatexSpanStyles(fontSizeEm: 2.488), // .reset-size6.size11
                 text: '1',
                 nodes: null),
-              KatexNode(
+              KatexSpanNode(
                 styles: KatexSpanStyles(fontSizeEm: 2.074), // .reset-size6.size10
                 text: '2',
                 nodes: null),
-              KatexNode(
+              KatexSpanNode(
                 styles: KatexSpanStyles(fontSizeEm: 1.728), // .reset-size6.size9
                 text: '3',
                 nodes: null),
-              KatexNode(
+              KatexSpanNode(
                 styles: KatexSpanStyles(fontSizeEm: 1.44), // .reset-size6.size8
                 text: '4',
                 nodes: null),
-              KatexNode(
+              KatexSpanNode(
                 styles: KatexSpanStyles(fontSizeEm: 1.2), // .reset-size6.size7
                 text: '5',
                 nodes: null),
-              KatexNode(
+              KatexSpanNode(
                 styles: KatexSpanStyles(fontSizeEm: 1.0), // .reset-size6.size6
                 text: '6',
                 nodes: null),
-              KatexNode(
+              KatexSpanNode(
                 styles: KatexSpanStyles(fontSizeEm: 0.9), // .reset-size6.size5
                 text: '7',
                 nodes: null),
-              KatexNode(
+              KatexSpanNode(
                 styles: KatexSpanStyles(fontSizeEm: 0.8), // .reset-size6.size4
                 text: '8',
                 nodes: null),
-              KatexNode(
+              KatexSpanNode(
                 styles: KatexSpanStyles(fontSizeEm: 0.7), // .reset-size6.size3
                 text: '9',
                 nodes: null),
-              KatexNode(
+              KatexSpanNode(
                 styles: KatexSpanStyles(fontSizeEm: 0.5), // .reset-size6.size1
                 text: '0',
                 nodes: null),
             ]),
         ]),
     ]);
 
-  static final mathBlockKatexNestedSizing = ContentExample(
+  static const mathBlockKatexNestedSizing = ContentExample(
```

Comment:
```
+  // TODO: Re-enable this test after adding support for parsing
+  //       `vertical-align` in inline styles. Currently it fails
+  //       because `strut` span has `vertical-align`.
+  //
+  // static const mathBlockKatexDelimSizing = ContentExample(
```

Similarly, use `skip: true` instead of commenting.

For this test, that requires a bit of setup to make this file's helpers and the "all content examples are tested" check handle the `skip:` argument. I've just pushed a small added commit at the end of this PR branch to do that:
f9a79d875 content test [nfc]: Enable skips in testParseExample and testParse

So please rebase that to earlier in the branch as needed, and then write:
```dart
  testParseExample(ContentExample.mathBlockKatexNestedSizing, skip: true);
```